src/cloud/trouble/error-codes.md

```diff
@@ -0,0 +1,194 @@
+| compile-di | Command `/bin/magento setup:di:compile` failed | 15 |
+| dump-autoload | Command `composer dump-autoload` failed | 16 |
+| run-baler | If baler configured but execution of `baler` command failed | 17 |
+| compress-static-content | Required utility wasn't found (timeout, bash) | 18 |
+| deploy-static-content | Command `/bin/magento setup:static-content:deploy` failed | 19 |
+| compress-static-content | Compression of static content failed | 20 | 
+| backup-data: static-content | Failed to copy static content into init directory | 21 |
+| backup-data: static-content | Failed to clean `./init/pub/static/` directory | 24 |
+| backup-data: writable-dirs | Failed to copy some writable directory into init directory | 22 |
+| general | Unable to create a logger object | 23 |
+| | Composer package was not found | 25 |
+
+### Deploy stage
+
+| Step | Description | Error code |
+| --- | --- | --- |
+| pre-deploy: cache | Wrong cache configuration (missed port or host) | 101 |
+| general | File `./app/etc/env.php` is not writable | 102 |
+| | Configuration isn't defined in schema.yaml | [103](#CONFIG_NOT_DEFINED) |
+| | Failed to parse .magento.env.yaml | 104 |
+| | Unable to read .magento.env.yaml | 105 |
+| | Unable to read .schema.yaml | 106 |
+| pre-deploy: clean-redis-cache | Failed to clean redis cache | 107 |
+| pre-deploy: set-production-mode | Command `/bin/magento maintenance:enable` failed | 108 |
+| validate-config | Wrong database configuration | 109 |
+| | Wrong session configuration | 110 |
+| | Wrong search configuration | 111 |
+| | Wrong resource configuration | 112 |
+| validate-config:elasticsuite-integrity | ElasticSuite is installed without available ElasticSearch service | 113 |
+| | ElasticSuite is installed but another search engine is used | 114 |
+| general | Database query execution failed | 115 |
+| install-update: setup | Command `/bin/magento setup:install` failed | 116 |
+| install-update: config-import | Command `app:config:import` failed | 117 |
+| general | Required utility wasn't found (timeout, bash) | 118 |
+| install-update: deploy-static-content	 | Command `/bin/magento setup:static-content:deploy` failed | 119 |
+| compress-static-content | Compression of static content failed | 120 |
+| deploy-static-content:generate | Cannot update deployed version | 121 |
+| clean-static-content | Failed to clean static content files | 122 |
+| install-update: split-db | Command `/bin/magento setup:db-schema:split` failed | 123 |
+| clean-view-preprocessed | Failed to clean var/view_preprocessed | 124 |
+| install-update: reset-password | Failed to update `/var/credentials_email.txt` file | 125 |
+| install-update: update | Command `/bin/magento setup:upgrade` failed | 126 |
+| clean-cache | Command `/bin/magento cache:flush` failed | 127 |
+| disable-maintenance-mode | Command `/bin/magento maintenance:disable` failed | 128 | 
+| install-update: reset-password | Enable to read reset password template | 129 |
```

Comment:
This one is a little confusing --  What is the error?  That the reset password template can't be read?